fabric-entity-tick-events-v1/src/main/java/net/fabricmc/fabric/impl/event/entity/EntityCascadingEventBridge.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.event.entity;
+
+import net.minecraft.entity.Entity;
+
+import net.fabricmc.fabric.api.event.Event;
+import net.fabricmc.fabric.api.event.v1.EntityTickCallback;
+
+public interface EntityCascadingEventBridge {
+	Event<EntityTickCallback<Entity>> getTickEvent();
```

Comment:
Should mixin interface methods have a fabric_ prefix to avoid conflicts?

Reply:
If it's not part of the API I would say definitely yes.

Reply:
Should have no issues as is